cfme/web_ui/expression_editor.py

```diff
@@ -34,8 +35,12 @@ def _expressions_root():
 # Buttons container
 buttons = Region(
     locators=dict(
-        commit="//img[@alt='Commit expression element changes']",
-        discard="//img[@alt='Discard expression element changes']",
+        commit=deferred_verpick(
+            {version.LOWEST: "//img[@alt='Commit expression element changes']",
+             '5.7': "//button[@title='Commit expression element changes']"}),
```

Comment:
Doe sdeferred verpick work here? Region should be able to process dict as verpick automatically anyways ... deferred_verpick is only for placing on classes.